geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLColumnsTest.scala

```diff
@@ -118,8 +119,8 @@ class SparkSQLColumnsTest extends Specification with LazyLogging {
     "map appropriate column types" in {
       val df = sc.sql(s"select * from ${sft.getTypeName}")
 
+      // note: bytes, list, map not supported
```

Comment:
Where are those not supported?  In Parquet or generally in Spark?

Reply:
not sure, this was just copied from the existing code. i would think bytes at least would be supported across them all... maybe we just never implemented it

Reply:
Yes for bytes and map. There is an ArrayType as well.

https://spark.apache.org/docs/2.4.7/api/scala/index.html#org.apache.spark.sql.types.package

Reply:
yeah I'm thinking that this refers to the attribute types in a simple feature type, and we just never implemented these.